docs/blog/arcade/intro-cloud-sync.md

```diff
@@ -0,0 +1,40 @@
+# Introducing Cloud Sync for MakeCode Arcade
+
+**Posted on August 12th, 2021 by [eanders-ms](https://github.com/eanders-ms)**
+
+**Note**: As of this writing, sign-in and cloud sync are only available in [MakeCode Arcade](https://arcade.makecode.com).
+
+Over the past year we've been working to make it possible to sign into MakeCode with a [Microsoft Account](https://aka.ms/AAdd6f8). When you sign into MakeCode, you make it possible for us to remember who you are, and this unlocks a bunch of exciting new possibilities. For example, it becomes possible for us to remember what you were working on and make your projects available wherever you are. In fact this is the first feature we've built for our signed in users, and we're calling it **Cloud Sync**.
+
+## What is Cloud Sync?
+
+Have you ever been working on a MakeCode project at school and wanted to take it home? In the past, users have found creative ways to move projects from device to device and browser to browser, and these methods will still work! But Cloud Sync makes this a seamless experience: when you sign into MakeCode, your cloud-saved projects are download to you automatically.
+
+Perhaps you've had the experience of returning to a lab computer only to realize the machine had been reset overnight, erasing your projects with it. Losing all your work can be devastating. Cloud Sync guards against this by continouously saving your work to the cloud as you edit your project.
+
+## Nice! How do I enable it?
+
+Cloud sync is enabled automatically when you sign into MakeCode. After signing in, your existing projects remain local, meaning they're not automatically transferred to the cloud all at once. To transfer an existing project to the cloud, just open it in the editor. The editor's auto-save feature will trigger a cloud save and voilà, your project is in the cloud. Look for the editor's new cloud save indicator:
```

Comment:
Might be good to add that when you start a new project when signed it automatically is saved to the cloud